src/simulator_core/entities/assets/asset_abstract.py

```diff
@@ -44,17 +44,18 @@ class AssetAbstract(ABC):
     """The output of the asset as a list with a dictionary per timestep."""
     solver_asset: BaseAsset
 
-    def __init__(self, asset_name: str, asset_id: str):
+    def __init__(self, asset_name: str, asset_id: str, geometry: any):
```

Comment:
I would make the geometry a default parameter. That will save you from adjusting the code over the whole repository. 

I would afterwards set the geometry for all assets with a development routine? 

@samvanderzwan what is your opinion?